src/Sylius/Behat/Element/Admin/ProductOption/FormElement.php

```diff
@@ -14,92 +14,57 @@
 namespace Sylius\Behat\Element\Admin\ProductOption;
 
 use Behat\Mink\Element\NodeElement;
-use Behat\Mink\Exception\ElementNotFoundException;
 use Sylius\Behat\Behaviour\ChecksCodeImmutability;
-use Sylius\Behat\Behaviour\NamesIt;
 use Sylius\Behat\Behaviour\SpecifiesItsField;
 use Sylius\Behat\Element\Admin\Crud\FormElement as BaseFormElement;
-use Webmozart\Assert\Assert;
 
 class FormElement extends BaseFormElement implements FormElementInterface
 {
-    use NamesIt;
     use SpecifiesItsField;
     use ChecksCodeImmutability;
 
-    public function addOptionValue(string $code, string $value): void
+    public function nameItIn(string $name, string $localeCode): void
```

Comment:
```suggestion
    public function setName(string $name, string $localeCode): void
```
Since we got rid of the trait, why not change the name as well?